### DISCUSSION TITLE
Import Common LUT Format (CLF)

### DISCUSSION THREAD

**Discussion by ssh4net** [videovillage/community#201] [Ideas]
Just bought Lattice and found even if it promised support for "All" LUT formats, it do not have support for ACES Common LUT Format.

As a dev i more than sure that basic support for CLF import can be not so hard to add, as Lattice already have all required controls to work with CLF 

Thank you

**Comment by gregcotten** [comment]
Hey not saying we shouldn't support CLF, but we certainly don't promise it supports "All" LUT formats as you have mentioned. We also have a [list of supported LUTs on the website](https://videovillage.com/lattice/).

**Reply by gregcotten** [reply]
Correction: we say "Every" which is essentially the same and clearly an absurd promise to begin with. I will amend the website.

**Reply by ssh4net** [reply]
Hah, no worries. Just CLF quite promising LUT format more flexible than others.
It already have perfect support in OCIO. And will be nice to have it in Lattice to test it authoring.
